client/me/help/help-contact-form/index.jsx

```diff
@@ -20,13 +21,14 @@ import FormTextInput from 'components/forms/form-text-input';
 import FormButton from 'components/forms/form-button';
 import SitesDropdown from 'components/sites-dropdown';
 import siteList from 'lib/sites-list';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
 
 /**
  * Module variables
  */
 const sites = siteList();
 
-module.exports = React.createClass( {
+const HelpContactForm = React.createClass( {
 	displayName: 'HelpContactForm',
```

Comment:
Looks like this was incorporated in 5ca23c9e822ad47299ee369b415b2cf172872cab (#8281). Thanks for catching @spentay !